lib/discourse_plugin_registry.rb

```diff
@@ -117,6 +118,10 @@ def self.register_auth_provider(auth_provider)
     self.auth_providers << auth_provider
   end
 
+  def self.register_mail_poller(mail_poller)
+    self.mail_pollers << mail_poller
+  end
```

Comment:
Just need a small test in https://github.com/discourse/discourse/blob/main/spec/lib/discourse_plugin_registry_spec.rb

Reply:
Thanks, I added the test for the empty set!